warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_stale_vehicle_positions.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    materialized='incremental',
+    incremental_strategy='insert_overwrite',
+    partition_by = {
+        'field': 'date',
+        'data_type': 'date',
+        'granularity': 'day',
+    },
+) }}
+
+{% if is_incremental() %}
+    {% set timestamps = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set max_ts = timestamps[0] %}
+{% endif %}
+
+WITH
+
+feed_guideline_index AS (
```

Comment:
One option is to follow [this design](https://github.com/cal-itp/data-infra/blob/main/warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__rt_https.sql) and build checks for all 3 rt feeds in one. I think I could make this update if you didn't want to.

Reply:
I looked into it and it turns out I agree with you that we should keep the feeds separately. I went ahead and added some missing file updates, edited a bit.